### ISSUE TITLE
[Accessibility] In high contrast mode the dropdown icons are not visible

### ISSUE THREAD

**Issue by jiaqi3** [ng-select/ng-select#2109]
**Describe the bug**
When applied window high contrast mode, the dropdown icon changed to a black box.

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. use hign constrast.
![image](https://user-images.githubusercontent.com/70375028/177723959-76b05204-1edc-4552-a483-ec2cba85b2f2.png)

**Expected behavior**
show the same icon in high constrast mode.

**Screenshots**
collapse:
![image](https://user-images.githubusercontent.com/70375028/177723176-17245199-bfa4-4a6f-981e-9773c7250201.png)
expand:
![image](https://user-images.githubusercontent.com/70375028/177723254-b45a7c16-9338-416e-895d-e0782372bcdf.png)
